### ISSUE TITLE
Kerosine newcomen pump with wick burner needs more transitions

### ISSUE THREAD

**Issue by HimitsuGato** [jasonrohrer/OneLifeData7#1181]
Accidentally removing the wick burner from the pump can be punishing for no reason.

Currently you can only add a kerosine wick to a newcomen pump when the well is full. At any point in refilling it -torn seal, dry kerosine, full boiler-  you can disassemble the wick burner from the with a simple misclick. This one way transition can create frustrating interractions, where passerbys will remove it accidentally, and then you have to run the whole pump with charcoal again to re-apply the wick burner.

**Comment by DougLefelhocz** [comment]
I agree on the annoyance.  I think Jason even understands the annoyance here.

[Possibly related comments though](https://github.com/jasonrohrer/OneLifeData7/issues/1113) (there's more links to follow)

>  Never made much sense before that you could only add the kero burner …

…to the wet pump. It also caused some complications, because the two wet pumps have different water levels, and the transitions needed to track that. Now you can only add the kero burner to the dry pump. You no longer need to fire it with charcoal before adding the burner. This makes a lot more sense.

[1](https://github.com/jasonrohrer/OneLifeData7/commit/2ea0bd0c608bc362d77a2d18190bef01494e4d91)

>  Revert, since this broke all other Newcommen machines that were part …

…of the same patter.

[2](https://github.com/jasonrohrer/OneLifeData7/commit/46d558fce18500b864c400ca300b4711404e1dfb)

From the 0th link

> Oh, so that's the reason the wick could only be added when they are wet!

The dry machines are all part of a pattern....

The other machines never get "wet", so that's the only spot that's "safe" to insert the wick. I knew there must be a reason for this.

The editor now supports comments on transitions, so I will add one to explain this!

**Comment by HimitsuGato** [comment]
I had no idea the pedigree on this bug. Perhaps an alternate solution would be to make it harder to remove the kerosine wick?

**Comment by Shadie116** [comment]
yuh es
